mfr/extensions/papaya/static/papaya.js

```diff
@@ -0,0 +1,2105 @@
+var PAPAYA_BUILD_NUM="1430",papayaLoadableImages=[];
```

Comment:
There are several different version of released code. Is this the smallest file that covers our need. (e.g. If their minimal version has what we need, we can use that one (1K+) instead of this (2K+).

Reply:
I think that is a question for product team. I demoed, and they appeared to like the added functionality.